sky/jobs/server/core.py

```diff
@@ -159,7 +159,8 @@ def launch(
     # pre-mount operations when submitting jobs.
     dag.pre_mount_volumes()
 
-    user_dag_str = dag_utils.dump_chain_dag_to_yaml_str(dag)
+    user_dag_str = dag_utils.dump_chain_dag_to_yaml_str(dag,
```

Comment:
Can we rename this variable, e.g. `user_dag_str_redacted`? I worry that a future change may assume this is the complete dag.
I guess this PR isn't actually changing the behavior here (it was already redacted), but may be good for the future. Doesn't need to block merge.